lhm.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest'
   s.add_development_dependency 'mocha'
   s.add_development_dependency 'rake'
-  s.add_development_dependency 'activerecord'
+  s.add_development_dependency 'activerecord', '~> 6.1.4'
```

Comment:
I'm a bit confused. Doesn't LHM depend on an ActiveRecord connection? Why is it only a development dependency? I guess this is more a general question and not specific to this PR.

Reply:
Yes, it does require acrtiverecord to use. However, for development we would require 6.1.4 and greater to ensure some consistency with the development (which has been a problem in the past)

Reply:
Couldn't we just check in the `Gemfile.lock`? I'm not sure why we put it in gitignore in the first place. The commit is just saying "No gemfile.lock for gems". I checked a few other Shopify gems and they all have it checked in. 🤔

Reply:
At one point LHM didn't depend on AR... or at least that was the intention. Not sure if that was ever successful.